htmlreport/tox.ini

```diff
@@ -1,12 +1,15 @@
 [tox]
-envlist=py26,py27,py32,pypy
+envlist = py26,py27,py32,pypy,pip
```

Comment:
I guess we no longer need the `py26` stuff. Also Python 2.7 needs to be officially deprecated (I filed https://trac.cppcheck.net/ticket/12682 about that).

Reply:
Sure, I can remove that. I have not found any usage of `tox` though in the CI scripts you have. Would you confirm if it is still in use?

Reply:
I have do not even have an idea what `tox` is...

If this is unused we should just remove it.

Reply:
If we keep the file please make sure it was moved within git. Currently it shows up as deleted/added.

Reply:
BTW Python 2.7 is deprecated now and the leftovers will be removed after the next release.

Reply:
It should be moved. At least commit history show it that way:
![image](https://github.com/danmar/cppcheck/assets/5689770/823afb90-6d9e-4f31-8116-d091382c6733)

Not sure why it shows up as removed and created under the "File changed" tab.

Reply:
I have encountered this before. It seems to be a bug or at least some quirk within git/GitHub - and a bad one. Fortunately it only occurred with unimportant files so far.

Reply:
It is a known GitHub issue: https://github.com/orgs/community/discussions/43177